tests/x11/multi_users_dm.pm

```diff
@@ -38,11 +38,22 @@ sub ensure_graphical_target {
 }
 
 sub restart_x11 {
-    ensure_multi_user_target;
-    ensure_graphical_target;
+    my ($self, $setnologin) = @_;
+    record_info('workaround', 'Graphical errors on cirrus when restarting X11 -> performing reboot (see bsc#1063060)');
+    type_string "reboot\n";
+    my $timeout = get_var('UEFI') ? 140 : 80;
+    if ($setnologin) {
+        $self->wait_boot(bootloader_time => $timeout, nologin => 1, reallynologin => 1);
+    }
+    else {
+        $self->wait_boot(bootloader_time => $timeout);
```

Comment:
We can combine these two branches in the single call by setting params to $setnologin
```perl
nologin => $setnologin, reallynologin => $setnologin
```